File: Symfony/src/BGR/Serrano/ProductoBundle/Entity/Paquete.php
<?php

namespace BGR\Serrano\ProductoBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use JMS\Serializer\Annotation\Type;
use JMS\Serializer\Annotation\Exclude;
/**
 * Paquete
 *
 * @ORM\Table()
 * @ORM\Entity(repositoryClass="BGR\Serrano\ProductoBundle\Entity\PaqueteRepository")
 */
class Paquete
{
    /**
     * @var integer
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     * @Type("integer")
     */
    private $id;

    /**
     * @var string
     *
     * @ORM\Column(name="codigo", type="string", length=255)
     * @Type("string")
     */
    private $codigo;

    /**
     * @var string
     *
     * @ORM\Column(name="estado", type="string", length=255)
     * @Type("string")
     */
    private $estado;


    /**
     * Get id
     *
     * @return integer 
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * Set codigo
     *
     * @param string $codigo
     * @return Paquete
     */
    public function setCodigo($codigo)
    {
        $this->codigo = $codigo;
    
        return $this;
    }

    /**
     * Get codigo
     *
     * @return string 
     */
    public function getCodigo()
    {
        return $this->codigo;
    }

    /**
     * Set estado
     *
     * @param string $estado
     * @return Paquete
     */
    public function setEstado($estado)
    {
        $this->estado = $estado;
    
        return $this;
    }

    /**
     * Get estado
     *
     * @return string 
     */
    public function getEstado()
    {
        return $this->estado;
    }




   /**
     * @ORM\ManyToOne(targetEntity="Presentacion") 
     * @ORM\JoinColumn(name="presentacion_id", referencedColumnName="id")
     * @Type("BGR\Serrano\ProductoBundle\Entity\Presentacion")
     */
    protected $presentacion;

   /**
     * @ORM\ManyToOne(targetEntity="Mix") 
     * @ORM\JoinColumn(name="mix_id", referencedColumnName="id")
     * @Type("BGR\Serrano\ProductoBundle\Entity\Mix")
     */
    protected $mix;

    public function setPresentacion(Presentacion $presentacion)
    {
        $this->presentacion = $presentacion;
    }

    public function getPresentacion()
    {
        return $this->presentacion;
    }
	public function getMix() {
		return $this->mix;
	}
	public function setMix($mix) {
		$this->mix = $mix;
		return $this;
	}
	

}

File: BGR/BGR-Data/src/main/java/ar/com/bgr/serrano/model/Paquete.java
package ar.com.bgr.serrano.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.Table;



@Entity(name="Paquete")
@Table(name="Paquete")
public class Paquete {

	@Id
	@GeneratedValue
	private int id;
	
	@Column(name="codigo")
	private String codigo;
	
	@Column(name="estado")
	private String estado;
	
    @Column(name="presentacion_id")
    private Integer presentacion_id;    
	
    @Column(name="mix_id")
    private Integer mix_id; 
    
	public Paquete(){}
	
	/**
	 * @return the id
	 */
	public int getId() {
		return id;
	}

	/**
	 * @param id the id to set
	 */
	public void setId(int id) {
		this.id = id;
	}

	/**
	 * @return the codigo
	 */
	public String getCodigo() {
		return codigo;
	}

	/**
	 * @param codigo the codigo to set
	 */
	public void setCodigo(String codigo) {
		this.codigo = codigo;
	}

	/**
	 * @return the estado
	 */
	public String getEstado() {
		return estado;
	}

	/**
	 * @param estado the estado to set
	 */
	public void setEstado(String estado) {
		this.estado = estado;
	}

	/**
	 * @return the presentacion_id
	 */
	public Integer getPresentacion_id() {
		return presentacion_id;
	}

	/**
	 * @param presentacion_id the presentacion_id to set
	 */
	public void setPresentacion_id(Integer presentacion_id) {
		this.presentacion_id = presentacion_id;
	}

	/**
	 * @return the mix_id
	 */
	public Integer getMix_id() {
		return mix_id;
	}

	/**
	 * @param mix_id the mix_id to set
	 */
	public void setMix_id(Integer mix_id) {
		this.mix_id = mix_id;
	}
    
	
}
